src/contextprotector/mcp_wrapper.py

```diff
@@ -272,7 +218,7 @@ async def read_resource(name: str) -> types.ReadResourceResult:
             except McpError as e:
```

Comment:
nit: rename `content` to `blob` to avoid shadowing, might be a bit easier to track dataflow? https://github.com/trailofbits/mcp-context-protector/pull/10/files?diff=split&w=0#diff-47bef02058efff999338815c354bb39a34a0d7b5e97f0a764a7e468d67afc873R197